src/lib.rs

```diff
@@ -90,6 +90,34 @@ fn get_num_cpus() -> usize {
     }
 }
 
+/// Returns the number of physical CPUs of the current machine.
+#[inline]
+pub fn get_physical() -> usize {
+    get_physical_num_cpus()
```

Comment:
I realised that afterwards myself. Failing at compile time seems to be more idiomatic.